src/gluonts/model/lstnet/_network.py

```diff
@@ -82,30 +84,33 @@ def __init__(
             "lstm",
         ], "`skip_rnn_cell_type` must be either 'gru' or 'lstm' "
         self.conv_out = context_length - kernel_size + 1
-        conv_skip = self.conv_out // skip_size
-        assert conv_skip > 0, (
-            "conv1d output size must be greater than or equal to `skip_size`\n"
+        self.conv_skip = self.conv_out // skip_size
+        assert self.conv_skip > 0, (
+            "conv2d output size must be greater than or equal to `skip_size`\n"
             "Choose a smaller `kernel_size` or bigger `context_length`"
         )
-        self.channel_skip_count = conv_skip * skip_size
-        self.skip_rnn_c_dim = channels * skip_size
+        self.channel_skip_count = self.conv_skip * skip_size
         self.dtype = dtype
         with self.name_scope():
-            self.cnn = nn.Conv1D(
+            self.cnn = nn.Conv2D(
                 channels,
-                kernel_size,
+                (num_series, kernel_size),
                 activation="relu",
-                layout="NCW",
-                in_channels=num_series,
+                layout="NCHW",
+                in_channels=1,
             )  # NCT
             self.cnn.cast(dtype)
             self.dropout = nn.Dropout(dropout_rate)
             self.rnn = self._create_rnn_layer(
-                channels, rnn_num_layers, rnn_cell_type, dropout_rate
+                rnn_num_cells, rnn_num_layers, rnn_cell_type, dropout_rate
             )  # NTC
             self.rnn.cast(dtype)
+            self.skip_rnn_num_cells = skip_rnn_num_cells
             self.skip_rnn = self._create_rnn_layer(
-                channels, skip_rnn_num_layers, skip_rnn_cell_type, dropout_rate
```

Comment:
Same discussion as above, referring to the [original code](https://github.com/laiguokun/LSTNet/blob/master/models/LSTNet.py#L22).

Reply:
see my previous comment, this method takes the hidden size of the RNN not the input size which I believe Mxnet infers. The input features coming from the CNN will of course be of size `channels`.